patches/tModLoader/Terraria/ID/ItemRarityID.cs

```diff
@@ -3,6 +3,8 @@ namespace Terraria.ID
 	/// <summary>Enumerates the values used with Item.rare</summary>
 	public static class ItemRarityID
 	{
+		/// <summary>Minus thirteen (-12)\nMaster: Fiery Red\nFlag: item.master</summary>
```

Comment:
I'm certain you did not mean to type `Minus thirteen (-12)`.
```suggestion
		/// <summary>Minus thirteen (-13)\nMaster: Fiery Red\nFlag: item.master</summary>
```